### ISSUE TITLE
"Error in as_base_type(x, as.list) : object 'rlang_get_attrs' not found" with render_graph

### ISSUE THREAD

**Issue by nilescbn** [rich-iannone/DiagrammeR#302]
Hello, 

I've be long wanting to learn the DiagrammeR package and finally got around to it this morning. Thank you for your work. The package will be quite useful. 

Attempting to use `render_graph()`, I'm getting the following error:

`"Error in as_base_type(x, as.list) : object 'rlang_get_attrs' not found"` 

This first happened with a graph I created with `create_graph()`. Yet the error also happens with the examples from the `render_graph()` help file. 

The problem I'm having is with the default "graph" output type. The "visNetwork" output type works as expected. 

For example, this code renders fine.

```r
create_graph() %>%
  add_balanced_tree(
    k = 2, h = 3) %>%
  render_graph(
    output = "visNetwork")
```

This will throw the error.

```r
create_graph() %>%
  add_balanced_tree(
    k = 2, h = 3) %>%
  render_graph()
```

I use R quite frequently for data analysis but my programming skills are novice. I say this because I'm not confident in my ability to differentiate a likely error in the code from something wrong with my configuration. 

I first got the error using RMarkdown in RStudio version 1.2.830. Yet the same behavior happens in the RGui as well.My sesssion info using the RGui is pasted below. If additional information would be helpful, I'm happy to provide it. 

```r
R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 16299)

Matrix products: default

locale:
[1] LC_COLLATE=English_United States.1252 
[2] LC_CTYPE=English_United States.1252   
[3] LC_MONETARY=English_United States.1252
[4] LC_NUMERIC=C                          
[5] LC_TIME=English_United States.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] bindrcpp_0.2.2   DiagrammeR_1.0.0

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.18       plyr_1.8.4         pillar_1.3.0       compiler_3.5.1    
 [5] RColorBrewer_1.1-2 influenceR_0.1.0   bindr_0.1.1        viridis_0.5.1     
 [9] tools_3.5.1        digest_0.6.15      jsonlite_1.5       viridisLite_0.3.0 
[13] gtable_0.2.0       tibble_1.4.2       rgexf_0.15.3       pkgconfig_2.0.2   
[17] rlang_0.2.2        igraph_1.2.2       rstudioapi_0.7     yaml_2.2.0        
[21] gridExtra_2.3      downloader_0.4     dplyr_0.7.6        stringr_1.3.1     
[25] htmlwidgets_1.2    hms_0.4.2          grid_3.5.1         tidyselect_0.2.4  
[29] glue_1.3.0         R6_2.2.2           Rook_1.1-1         XML_3.98-1.13     
[33] clipr_0.4.1        readr_1.1.1        purrr_0.2.5        tidyr_0.8.1       
[37] ggplot2_3.0.0      magrittr_1.5       scales_1.0.0       htmltools_0.3.6   
[41] assertthat_0.2.0   colorspace_1.3-2   brew_1.0-6         stringi_1.2.4     
[45] visNetwork_2.0.4   lazyeval_0.2.1     munsell_0.5.0      crayon_1.3.4 
```

**Comment by nilescbn** [comment]
I appreciate the quick reply. 

I forgot to mention it but I did try that. I tried installing GitHub version again, this time restarting everything and uninstalling the package first. No luck, same behavior.  

But I got it working now. The first example from the `render_graph()` help file now produces the graph in RStudio's Viewer.

In case this is helpful to anyone else, it could have been a `rlang` issue. When you mentioned it, I remembered I updated to a new version of `rlang` recently (maybe even just this morning...).  As my session info shows, I was on v2.2. I tried installing the GitHub version of `rlang` to see if that helped. Somehow that installed `rlang` v 2.1.9000, which `DiagrammeR` did not like...

Then I reinstalled `rlang` again, back to v2.2, and `render_graph()` worked as advertised.

Thanks again.
